### DISCUSSION TITLE
matter bridged devices not showing on google home

### DISCUSSION THREAD

**Discussion by sherwoac** [arendst/Tasmota#20776] [Q&A] [answered]
Problem: 
I can't see my matter bridged devices on google home.

Back story:
I've got an ESP32 plug with running Tasmota (v13.4), paired with matter in google home (hosted on nest mini 2), runs fine, toggles - prior to adding any bridge devices. 

Adding extra devices (hosted on ESP8266 w/ v13.4) via the bridge `Auto-configure remote Tasmota` shows them locally:
![image](https://github.com/arendst/Tasmota/assets/11062064/586a5da9-1682-4c9f-ab68-ae122f99757b)
..and:
MTRINFO:
```
19:12:44.421 CMD: Grp 0, Cmd 'MTRINFO', Idx 1, Len 0, Pld -99, Data ''
19:12:44.440 MQT: tele/tasmota_6EB108/RESULT = {"MtrInfo":{"Ep":2,"Name":"cabinet plug","Power":1}}
19:12:44.455 MQT: tele/tasmota_6EB108/RESULT = {"MtrInfo":{"Ep":43,"Name":"speakers","Power":1}}
19:12:44.468 MQT: tele/tasmota_6EB108/RESULT = {"MtrInfo":{"Ep":44,"Name":"overhead","Power":0}}
19:12:44.484 MQT: tele/tasmota_6EB108/RESULT = {"MtrInfo":{"Ep":45,"Name":"uplight","Power":0}}
19:12:44.511 MQT: tele/tasmota_6EB108/RESULT = {"MtrInfo":{"Ep":46,"Name":"USB","Power":0}}
19:12:44.531 MQT: stat/tasmota_6EB108/RESULT = {"MtrInfo":"Done"}
```
but they don't show up in google home.

If I start the other way:
- remove the device from google home
- reset the bridge via `Reset all and Auto-discover`
- add the devices to the bridge again as above
- _then_ try to add the matter device to google home
- .... it just hangs during matter pairing.

Any suggestions welcome.

**Comment by s-hadinger** [comment, marked as answer]
I will try on my side. Adding endpoints is supposed to be detected by the controller

**Comment by sherwoac** [comment]
actually I figured it out, I had to restart google home device to see bridged devices for some reason
![Screenshot_20240221-191307](https://github.com/arendst/Tasmota/assets/11062064/3ee37d45-be0e-4f40-9210-5588ec125ced)
